app/core/api/thang-types.coffee

```diff
@@ -0,0 +1,7 @@
+fetchJson = require './fetch-json'
+utils = require 'core/utils'
+
+module.exports = {
+  getAll: (options) ->
+    return fetchJson('/db/thang.type', options)
+}
```

Comment:
This needs to be checked for whether the options are coming through correctly. May be related to HeroSelectModal returning too many things.